### ISSUE TITLE
ActivityID is displayed as GUID

### ISSUE THREAD

**Issue by hwmland** [microsoft/perfview#1153]
I created small example for Activity. My problem is that ActivityID is displayed as GUID, not as nice 'path' like //1/3 Activity itself works - duration is calculated correctly even for nested activities.
Am I doing something wrong or behavior was changed?

PerfView version 2.0.54 (both 32b and 64b)
Test app: testet on .NET framework 4.6 and 4.7.2, both 32b and 64b
Thanks

**Comment by brianrob** [comment]
@hwmland, which view are you looking at in PerfView where you see the ID as a GUID?  Also, can you share the code that you're using to create the Activity?

**Comment by hwmland** [comment]
Hello Brian,
that is really bizzare. I'm not able to reproduce it anymore. 
I created very simple example (with 2 start/stop task pairs) and the same simple test application (just creation of some task inside another task and some CPU consumming loops). GUID was displayed in ActivityID column in events view. I make several tests, try to search internet - no success, so I reported this issue and let it be.
Today I returned ETW. I wanted to callstack on my events, found method how to switch them on using PerfView - and from this very moment ActivityID is correct (e.g. //1//13/1 and so on). I did not touch sources in meantime. I'm not generating callstack anymore, I even tried to reboot PC - and I'm not able to reproduce 'incorrect' behaviour anymore. Strange, but now it works for me, so I'll close this issue.
